Compiling from source - zlib not found with recent Cmake and ancient findZlib module

**Describe the bug**
Compiler can not find zlib in recent revisions / macOS version upgrades and on other platforms such as emscripten, linux
```
 Shared libraries disabled
-- Looking for ZLIB...
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2") 
-- Checking for module 'zzip-zlib-config'
--   No package 'zzip-zlib-config' found
-- Could not locate ZLIB
CMake Error at CMakeLists.txt:494 (message):
  Build configured with -DASSIMP_BUILD_ZLIB=OFF but unable to find zlib


-- Configuring incomplete, errors occurred!
```

emscripten:
```
configure: cmake -G "Unix Makefiles" -DBUILD_SHARED_LIBS=OFF -DASSIMP_BUILD_TESTS=0 -DASSIMP_BUILD_SAMPLES=0 -DASSIMP_BUILD_3MF_IMPORTER=0 "-DCMAKE_C_FLAGS=-O3 -DNDEBUG -pthread" "-DCMAKE_CXX_FLAGS=-O3 -DNDEBUG -pthread" .. -DCMAKE_TOOLCHAIN_FILE=/emsdk/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake -DCMAKE_CROSSCOMPILING_EMULATOR=/emsdk/node/14.18.2_64bit/bin/node;--experimental-wasm-threads
-- Shared libraries disabled
-- Looking for ZLIB...
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.6.3") 
-- Checking for module 'zzip-zlib-config'
--   Package 'zzip-zlib-config', required by 'virtual:world', not found
-- Could not locate ZLIB
CMake Error at CMakeLists.txt:494 (message):
  Build configured with -DASSIMP_BUILD_ZLIB=OFF but unable to find zlib


-- Configuring incomplete, errors occurred!
See also "/src/apothecary/build/assimp/build_emscripten/CMakeFiles/CMakeOutput.log".
emcmake: error: 'cmake -G "Unix Makefiles" -DBUILD_SHARED_LIBS=OFF -DASSIMP_BUILD_TESTS=0 -DASSIMP_BUILD_SAMPLES=0 -DASSIMP_BUILD_3MF_IMPORTER=0 "-DCMAKE_C_FLAGS=-O3 -DNDEBUG -pthread" "-DCMAKE_CXX_FLAGS=-O3 -DNDEBUG -pthread" .. -DCMAKE_TOOLCHAIN_FILE=/emsdk/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake -DCMAKE_CROSSCOMPILING_EMULATOR=/emsdk/node/14.18.2_64bit/bin/node;--experimental-wasm-threads' failed (returned 1)
```

**To Reproduce**
Steps to reproduce the behavior:
Build recent version master 

**Expected behavior**
finds zlib and compiles

**Platform (please complete the following information):**
 - iOS / emscripten 

 - Version master

**Additional context**
Workflow and build runs can be observed failing here:
iOS:
(https://github.com/openframeworks/apothecary/actions/runs/4696636258/jobs/8326906448?pr=231)
Emscripten:
https://github.com/openframeworks/apothecary/actions/runs/4696636243/jobs/8326905961?pr=231


On my Macbook updated Xcode tools to latest, brew install zlib, brew link zlib. No dice. 
macOS target compiles however

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -138,7 +138,7 @@ IF (WIN32)
 ELSE()
   OPTION( ASSIMP_BUILD_ZLIB
     "Build your own zlib"
-    OFF
+    ON
   )
 ENDIF()
 

